CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.6.0
+* [#104](https://github.com/cohere-ai/cohere-python/pull/96)
+    * Remove experimental `moderate` api
```

Comment:
Would this be a major semver bump, since it is breaking with past versions that use `moderate`?

Reply:
Oh unless we don't consider it breaking because the endpoint was experimental 🤔 -- pls disregard comment if so.

Reply:
We haven't formalized it, but unofficially we talk about experimental APIs as pre-beta. That means that an experimental api can be pulled at any point without a version change